### PR TITLE
index.rst: add hint

### PR DESCRIPTION
**Pull request by DumAdrian**
Closes: https://github.com/coala/documentation/issues/278

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -133,6 +133,11 @@ To programmatically access coala's functionality, use ``coala-json`` or
 binaries, among with ``coala-ci``, are noninteractive and thus suitable for
 continuous integration.
 
+.. Hint::
+
+    Use coala-json and coala-format!
```
Comment (Nosferatul) [inline index.rst]: Maybe something more specific?

Comment (Nosferatul) [inline index.rst]: Also write description in the body of commit message

Comment (sils) [inline index.rst]: yeah this isn't sufficient

**Comment by sils** [comment]
please read about the commit message style at coala.io/commits

**Comment by sils** [comment]
be more specific in the message and in the rst file please, this isn't really helping anyone I think

**Comment by sils** [comment]
(not trying to intimidate you btw.)

**Comment by DumAdrian** [comment]
Hei! I don't really understand what this hint is supposed to say. I wrote what it was specified in the issues.

**Comment by Adrianzatreanu** [comment]
closing due to high inactivity

**Pull request closed without merging**